Test-Source/NOTICE

```diff
@@ -0,0 +1,3 @@
+.NET Roslyn
```

Comment:
Why is the Notice added to Test-Source?
Is is not meant to be included in the package?

Reply:
Yes it's a mistake

Reply:
Will it automatically be included in the publish step?